src/script/components/RichTextEditor/plugins/GlobalEventsPlugin.tsx

```diff
@@ -81,9 +82,9 @@ export function GlobalEventsPlugin({onShiftTab, onArrowUp, onEscape, onBlur}: Gl
         KEY_ESCAPE_COMMAND,
         () => {
           onEscape();
-          return true;
+          return false;
         },
-        COMMAND_PRIORITY_LOW,
+        COMMAND_PRIORITY_EDITOR,
```

Comment:
This is a fix that will prevent canceling a reply when hitting `esc` to cancel the emoji/mention picker